grails-app/migrations/0.8.x/changelog-2022-06-08-1200-add-missing-roles.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <changeSet author="DariuszRodzewicz" id="0806202212000-0">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="0">select count(*) from role where role_type = "ROLE_PRODUCT_MERGE"</sqlCheck>
+    </preConditions>
+    <insert tableName="role">
+      <column name="id" value="6"/>
```

Comment:
Since IDs are strings, let's just make the id the same as the role type ... i.e. ROLE_PRODUCT_MANAGER.